mmv1/products/netapp/product.yaml

```
@@ -0,0 +1,28 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Product
+name: Netapp
+display_name: Netapp
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://netapp.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://netapp.googleapis.com/
```

```suggestion
    base_url: https://netapp.googleapis.com/v1beta1/
```

I think this should resolve the issue we see earlier. Sorry I was reading the error wrong in our meeting. The 404 returned for post request but not get request. I suspect that the local test works as it's tested in the GA provider only, and this error is only happened in the beta providers where our presubmit tests run